src/Compiler/Checking/CheckComputationExpressions.fs

```diff
@@ -81,7 +81,8 @@ let (|JoinRelation|_|) cenv env (expr: SynExpr) =
 
     | _ -> None
 
-let elimFastIntegerForLoop (spFor, spTo, id, start, dir, finish, innerExpr, m) = 
+let elimFastIntegerForLoop (spFor, spTo, id, start, dir, finish, innerExpr, m: range) =
+    let m = m.MakeSynthetic()
     let pseudoEnumExpr = 
         if dir then mkSynInfix m start ".." finish
```

Comment:
It feels like this use of `m` here and on next line is just wrong.  The range for the `..` expression should be `unionRanges start.Range finish.Range`?

Reply:
I think the best option is to cover `..` only, if we have its range in the tree.

But if we make the range synthetic, it'll be ignored by the most of the things anyway, similar to other rewritten expressions.

Reply:
In this case it's about cases like this:
```fsharp
let a = []
let _ = seq {
    for _ = 1 to 16 do yield! a
}
```
We don't have any `..` here, so I'm not sure what would be a better range to produce?

Reply:
The range over `1 to 16` or `1 downto 16` ?